docs/CLI/index.md

```diff
@@ -10,30 +10,40 @@ Install `oras` using [Homebrew](https://brew.sh/):
 brew install oras
 ```
 
+> Note: Homebrew only has ORAS v0.16.0 available to be used so far. It will be updated when ORAS v1.0.0 is released in Mar 2023.
+
 ## Release artifacts
 
-Install from the latest [release artifacts](https://github.com/oras-project/oras/releases):
+Install ORAS from the latest [release artifacts](https://github.com/oras-project/oras/releases):
 
 ### Linux
 
+If you want to install ORAS on an AMD64-based Linux machine, run the following command:
+
 ```bash
 curl -LO https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_linux_amd64.tar.gz
 mkdir -p oras-install/
 tar -zxf oras_1.0.0-rc.1_*.tar.gz -C oras-install/
-mv oras-install/oras /usr/local/bin/
+sudo mv oras-install/oras /usr/local/bin/
 rm -rf oras_1.0.0-rc.1_*.tar.gz oras-install/
 ```
 
+> Note: If you want to install ORAS on an ARM64-based Linux machine, you can download it from `https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_linux_arm64.tar.gz`.
+
 ### macOS
 
+If you want to install ORAS on an ARM64-based machine, run the following command:
```

Comment:
Apple uses specific terms for their products. See https://support.apple.com/en-us/HT211814
```suggestion
If you want to install ORAS on a Mac computer with Apple silicon, run the following command:
```